CatNote.Tests/Services/GenericServiceTests.cs

```diff
@@ -0,0 +1,168 @@
+﻿using AutoMapper;
+using CatNote.BLL.Models;
+using CatNote.BLL.Services;
+using CatNote.DAL.Entities;
+using CatNote.DAL.Interfaces;
+using CatNote.Tests.Services.DataForTests;
+using FluentAssertions;
+using Moq;
+using Xunit;
+
+namespace CatNote.Tests.Services;
+
+public class GenericServiceTests
+{
+    private readonly Mock<IGenericRepository<UserEntity>> _mockGenericRepository;
+    private readonly Mock<IMapper> _mockMapper;
+    private readonly GenericService<UserModel, UserEntity> _userService;
+
+    public GenericServiceTests()
+    {
+        _mockGenericRepository = new Mock<IGenericRepository<UserEntity>>();
+        _mockMapper = new Mock<IMapper>();
+        _userService = new GenericService<UserModel, UserEntity>(_mockMapper.Object, _mockGenericRepository.Object);
+    }
+
+    [Fact]
+    public async Task Create_CreateNewUser_CallCreateMethodOfRepositoryAndReturnUserModel()
```

Comment:
Read about GivenWhanThen

Reply:
@kalachck 
Given-When-Then (GWT) is a semi-structured way to write down test cases

Reply:
![image](https://github.com/Finka95/CatNote/assets/103589743/0a17fbb4-ea96-4d0b-bbd7-e84ed4ffe35e)